motor:
  hardware_interface:
     loop_hz: 10 # hz
     joints:
       - joint1